.github/workflows/pr_tests.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         # Run tests from integration_tests sub dir
         working-directory: ./integration_tests
     strategy:
+      fail-fast: false
```

Comment:
Would adding a `timeout-minutes: 45` be good to stop it altogether in under `jobs` in case there are long running queries? I thought there is already some limitation built into these.

Reply:
I was more thinking around if you do another push currently it will let the previous commit finish running, I don't think the timeout is needed as redshift just takes a long time, but isn't getting stuck